packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1773,6 +1774,12 @@ class Scaffold extends StatefulWidget {
   /// By default, the drag gesture is enabled.
   final bool endDrawerEnableOpenDragGesture;
 
+  /// Specifies if the Scaffold should register with the enclosing
+  /// [ScaffoldMessenger] to receive [SnackBar]s.
+  ///
+  /// Defaults to true.
+  final bool registerWithMessenger;
```

Comment:
Might be better to use the whole "ScaffoldMessenger" name and to come up with a name that reflects what's important from the developer's perspective: that we're disabling this Scaffold's ScaffoldMessenger (or that it's "declining snackbars" or something).